source/fem-dem/gls_vans.cc

```diff
@@ -228,6 +228,8 @@ void
 GLSVANSSolver<dim>::calculate_void_fraction(const double time,
                                             bool         load_balance_step)
 {
+  announce_string(this->pcout, "Void Fraction");
```

Comment:
Is the string in terminal to flag that it is executing the void fraction calculation or to print void fraction information?  That's a big unclear to me. 
If option 1, why it is printed as a header?
If option 2, why not at the end of cfd time step

Reply:
It's the string that indicates that the void fraction is being calculated.
The void calculation is calcualted before the VANS equations are solved, not after, hence why it is not at the end of the CFD time-step.
The "header" template is the same as the multiphysics environment. When a Physics is being solved for, it is first announced, then solved (if you have verbosity to on, you see the steps).

Reply:
Ok so it's option 1, no prob :)